inexor/ui/CefSubsystem.cpp

```diff
@@ -42,7 +43,14 @@ void CefSubsystem::tick() {
 
 void CefSubsystem::initialize(int argc, char **argv)
 {
-    spdlog::get("global")->info("CefSubsystem::initialize() --> CefInitialize()");
+    if (argc < 2) return;
+    std::string instance_id = argv[1];
+    std::string host = (argc > 2) ? argv[2] : "localhost";
```

Comment:
i think we should save this setting in the inexortree (remember: we already have it at this point of initialization)

Reply:
Why not in this PR? This is an ugly hack imo, while we can do it cleanly with less code.

Reply:
I mean ugly hack, since we tried to strip down any useless arguments (program options), and here we add them again without need.

Just add
```
SVARP(cef_server_hostname, "localhost");
VARP(cef_server_port, 31416);
```
and use them like regular variables after modifying them in flex :)